db/seeds.rb

```diff
@@ -1,7 +1,54 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+
+5.times do
+  user = User.new(
+    email:    Faker::Internet.email,
+    password: Faker::Lorem.characters(10)
+  )
+  user.skip_confirmation!
+  user.save!
+end
+users = User.all
+ 
+# Create Wikys
+50.times do
+  Wiky.create!(
+    user: users.sample,
+    title:  Faker::Lorem.sentence,
+    body:   Faker::Lorem.paragraph,
+    created_at: rand(2.hours .. 1.year).ago
+  )
+end
+wikys = Wiky.all
+
+# Create an admin user
+admin = User.new(
+  email:    'admin@example.com',
+  password: 'helloworld',
+  role:     'admin'
+)
+admin.skip_confirmation!
+admin.save!
+
+# Create an premium user
+admin = User.new(
+  email:    'premium@example.com',
+  password: 'helloworld',
+  role:     'premium'
+)
+admin.skip_confirmation!
+admin.save!
+
+# Create an standard user
+admin = User.new(
```

Comment:
`admin` is being used in `seeds.rb` for non-admin users, consider making this clearer